docs/developer/getting-started/terminology.md

```diff
@@ -1,6 +1,10 @@
 # Terminology
 
-This is part of the developer [getting started guide](./README.md). 
+This is part of the developer [getting started guide](./README.md).
+
+A longer list of terminology for onboarding new hires is in the [Rancher Engineering Confluence site.](https://confluence.suse.com/display/EN/Onboarding+and+New+Hire+Glossary)
```

Comment:
The confluence document is not available externally - I'd remove it for now.

Reply:
removed